### PR TITLE
Factored out generation of deploy-id and added random city name

### PR DESCRIPTION
**Pull request by jsuresh**
The latter is useful for talking about deploys between humans.

### CODE REVIEW

File: common.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import (
+	"fmt"
+	"rand"
+	"time"
+)
+
+func NewDeployId() string {
+	t := time.Now().UTC()
+	rand.seed(t.UnixNano())
```
Comment (soulplant) [inline common.go]: This doesn't need to be here.

File: adjectives.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
+package main
+
+// Random names to prefix to deploy ids.
+// Makes it easier to talk about some specific deploy
+
+var ADJECTIVES = []string{
```
Comment (soulplant) [inline adjectives.go]: Nice!

File: common.go (hunk 1 above)
Comment (jsuresh) [inline common.go]: We don't seed Random anywhere else? Should I just move this to main?

Comment (soulplant) [inline common.go]: Yeah, otherwise anywhere else we use randomness we would have to seed as well, otherwise we wouldn't be sure that it's been seeded yet.

**Comment by soulplant** [comment]
LGTM

File: common.go (hunk 1 above)
Comment (jsuresh) [inline common.go]: Of course - was being dumb. Fixed.

**Pull request merged**